neuro_flow/storage.py

```diff
@@ -136,7 +148,7 @@ async def create_bake(
         project: str,
         batch: str,
         config_name: str,
-        config_content: str,
+        configs: Iterable[Tuple[RemotePath, str]],
```

Comment:
please relax the type: `Iterable` -> `Sequence`.
It is still read-only but points on a container.
I believe, iterable is good for a function argument but looks a little weird as a data structure.

Reply:
Create a separate dataclass instead of `Tuple[RemotePath, str]`.
Just in case.